verify-scripts.js

```
@@ -0,0 +1,48 @@
+console.log('----------------------------------');
+
+// Verify that the start script does NOT include the test mode flag (runs in real mode)
+if (packageJson.scripts.start && !packageJson.scripts.start.includes('--test-mode')) {
+  console.log('✅ npm start correctly runs in REAL mode (no test flag)');
+} else {
+  console.log('❌ npm start script is missing or incorrectly includes test mode flag');
+}
+
+// Verify that start:mock includes the test mode flag
+if (packageJson.scripts['start:mock'] && packageJson.scripts['start:mock'].includes('--test-mode')) {
+  console.log('✅ npm run start:mock correctly runs in MOCK mode (with test flag)');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)